packages/textarea/src/css/index.js

```diff
@@ -17,27 +24,25 @@ export default {
     minHeight: '104px',
     width: '100%',
     borderRadius: '2px',
-    background: colors.bone,
+    background: colorsBackgroundLight[3],
     fontSize: type.fontSizeSmall,
     lineHeight: type.lineHeightStandard,
     fontWeight: type.fontWeightBook,
-    color: colors.gray03,
+    color: colorsTextIcon.HighOnLight,
     padding: `${layout.spacingXSmall} ${layout.spacingMedium}`,
-    border: 'none',
+    border: `1px solid ${colorsBorder.lowOnDark}`,
 
-    '&:focus': { outline: 'none' }
+    '&:focus': { outline: 'none' },
+    '&::placeholder': { color: colorsTextIcon.lowOnLight }
   },
   [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
+    border: `1px solid ${colorsBorder.highOnLight}`,
     '&:focus': { border: '1px solid transparent' }
   },
 
   [`.psds-text-area__field--error.psds-theme--${themeNames.light}`]: {
     border: '1px solid transparent'
   },
-  [`.psds-text-area__field.psds-theme--${themeNames.light}`]: {
-    background: colors.white,
```

Comment:
@danethurber Did you still want background: white?  I'm not seeing this transferred in the refactor.

Reply:
uh oh....i'll fix

Reply:
looks like i added a white background to the field regardless of theme. so this change is correct